src/Silex/Provider/ValidatorServiceProvider.php

```diff
@@ -46,7 +48,21 @@ public function register(Container $app)
         };
 
         $app['validator.mapping.class_metadata_factory'] = function ($app) {
-            return new LazyLoadingMetadataFactory(new StaticMethodLoader());
+            $loader = null;
```

Comment:
If instead of `null` you set it to `new StaticMethodLoader` here you can remove the whole `else` class later.